scripts/docs/templates/integrations_doc.txt

```diff
@@ -0,0 +1,157 @@
+
```

Comment:
Call this `integrations_docs.md.jinja2` to make it clear it is a template for a markdown file.

Reply:
updated